tooling/nargo_cli/tests/execution_success/trait_multi_module_test/src/module5.nr

```diff
@@ -0,0 +1,2 @@
+struct MyStruct5 {
+}
```

Comment:
We don't need modules 4, 5, or 6 here. The test will be simpler if we define another struct type in module 2 and include the impl that is currently in module 6 in module 3 along with the first.

Reply:
That's not entirely true. During development I discovered I could get weird parser errors for modules that start with "impl", followed by path, instead of modules that start with "use". For this reason, I wanted to keep a module in the test that starts with "impl", without any "use" before that. But you're right that modules 4 and 5 are not needed. Maybe I will remove them from the test and perhaps add comment that explains why we want a module that starts with "impl".

Reply:
Can you file an issue for the `impl` parse error instead?